core/tx_pool.go

```diff
@@ -339,17 +341,6 @@ func (pool *TxPool) Pending() (map[common.Address]types.Transactions, error) {
 	pool.mu.Lock()
 	defer pool.mu.Unlock()
```

Comment:
Ah, strike that, you need to know that `m.items` does not change during the loop.